pom.xml

```diff
@@ -150,8 +150,8 @@
     <lombok.version>1.16.22</lombok.version>
     <lz4.version>1.3.0</lz4.version>
     <mockito.version>2.22.0</mockito.version>
-    <netty.version>4.1.31.Final</netty.version>
-    <netty-boringssl.version>2.0.19.Final</netty-boringssl.version>
+    <netty.version>4.1.32.Final</netty.version>
```

Comment:
Why not 4.1.33?
We have much time for testing

Reply:
@eolivelli I am using a version that gRPC is using.